dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
+        if (index===0) {
+          return currentValue
+        }
+        if(currentValue.attr&&currentValue.visible){
```

Comment:
But what if the first column is not bulk, but doesn't have an attr? You're blindly including the first column always.  Also the bulk column can be anywhere, not just the first position.. 

> All columns should have attr or title

Okay, that sounds reasonable, yes.. where is this code checking title though?